### PR TITLE
fix(AttentionBox): fix layout and typography

### PR DESCRIPTION
**Pull request by talkor**
Design request
https://monday.monday.com/boards/3532714909/pulses/5661772248

### CODE REVIEW

File: src/components/AttentionBox/AttentionBox.module.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .attentionBox {
   position: relative;
-  padding: var(--spacing-medium);
+  padding: 12px var(--spacing-medium);
```
Comment (talkor) [inline src/components/AttentionBox/AttentionBox.module.scss:3]: The 12px is intentional

Comment (YossiSaadi) [reply src/components/AttentionBox/AttentionBox.module.scss:3]: noooo 😟

**Review by YossiSaadi — APPROVED** [review]
It's weird that Chromatic didn't fail 🤔

**Comment by YossiSaadi** [comment]
btw, now the close button looks positioning weird

**Comment by talkor** [comment]
@YossiSaadi Yeah the close button is relative and not aligned to the title, it's intentional
And Chromatic failed, I fixed it in a previous build in this branch

**Pull request merged**